.github/workflows/goreleaser.yml

```diff
@@ -0,0 +1,33 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
nitpick, but if proper regex is possible in this field, `v\d+\.\d+\.\d+` would be better